src/app/group/group.controller.js

```diff
@@ -1,18 +1,29 @@
 'use strict';
 
 angular.module('studyBuddy')
-    .controller('GroupCtrl', function($firebaseObject, $firebaseArray){
-        var ref = new Firebase('https://study-buddy.firebaseio.com/groups');
-        this.obj = $firebaseArray(ref);
-        console.log("banana");
-        this.groupsObject = {};
+    .controller('GroupCtrl', function($firebaseObject, $firebaseArray, Auth){
+//    this.login = Auth.fbLogin;
+    this.logout = Auth.logout;
+    var self = this;
+    Auth.onAuth(function(user){
+        self.user = user;
         
-        this.newGroup = {
+    });
+//    var myGroup = self.user.child('groupInfo');
+//    console.log(myGroup);
+    var ref = new Firebase('https://study-buddy.firebaseio.com/users/' + self.user.$id);
+    var ref2 = $firebaseObject(new Firebase('https://study-buddy.firebaseio.com/users/' + self.user.$id))
+    ref2.$loaded()
+    .then(function(data){
+        self.name = data.name
+    })
+    this.obj = $firebaseArray(ref);
```

Comment:
I detect much confusion here... when `onAuth` completes `self.user` will be the user object for the authenticated user. Attempting to load that user as a `$firebaseArray` doesn't really make sense. I can see what you're trying to do with it afterward... maybe you want a `groups` property of that user object loaded as a `$firebaseArray` instead?